sally.yaml

```diff
@@ -5,6 +5,8 @@ url: go.uber.org
 packages:
     atomic:
         repo: github.com/uber-go/atomic
+    cadence:
```

Comment:
is it ok that cadence points to cadence-client, i.e. if we wanted to oss rewrite the server later would there be confusion?